### ISSUE TITLE
[Accessibility] Missing content descriptions for interactive buttons

### ISSUE THREAD

**Issue by balaraz** [mostafaalagamy/Metrolist#2000]
### Checklist

- [x] I am able to reproduce the bug with the [latest debug version](https://github.com/mostafaalagamy/Metrolist/actions).
- [x] I've checked that there is no open or closed issue about this bug.
- [x] This issue contains only one bug.
- [x] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

1. Enable TalkBack.
- 
  2. Navigate through the main library or playlist screens.
  3. Move focus to the action buttons (e.g., icons for "Revers sorting", "View (grid/list)", etc.).
- 
  2. Open full screen player.
  3. Move focus to the action buttons (e.g., icons for "Play", "Prev/next song", "Like song", etc.).

### Expected behavior

Every interactive UI element must have a meaningful label (e.g., "More options", "Play", "Add to favorites").

### Actual behavior

Observation: TalkBack announces "Unlabeled".

### Logs

There is no need, this is permanent behavior.

### Metrolist version

12.7.1

### Android version

15
